package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "groceries",
+  "version": "1.0.0",
+  "description": "The official repository for the 'groceries' project.",
+  "main": "gulpfile.js",
+  "browserslist": [
+    "defaults",
+    "not IE 11",
+    "not IE_Mob 11"
+  ],
```

Comment:
here is the browsers list.  now part of package.json.  i suppose it could say `"last 4 versions"` if we want it to.